src/graphics/vulkan/vulkan_swapchain.cpp

```
@@ -24,7 +25,8 @@ namespace {
 namespace canyon::graphics::vulkan {
     Swapchain::Swapchain(SurfaceContext& context, RenderPass& renderPass, VkSurfaceKHR surface, VkExtent2D extent)
```

Clang (14.0.6)

[error] constructor does not initialize these fields: m_extent, m_vkSwapchain

(cppcoreguidelines-pro-type-member-init,-warnings-as-errors)